### ISSUE TITLE
Interledger Protocol: Explain Like I'm Five (ELI5)

### ISSUE THREAD

**Issue by fjx2000** [interledger/rfcs#444]
# Asset Exchange & Connectors

What's particularly useful "about about" ILP payments......

*Should remove one "about" from the sentence above.*

**Comment by dora-gt** [comment]
Where is it?

**Comment by fjx2000** [comment]
https://github.com/interledger/rfcs/blob/master/0020-explain-like-im-five/0020-explain-like-im-five.md
(you should be able to find it by searching "about about")

"Asset Exchange & Connectors

What's particularly useful ####about about##### ILP payments, is that the kind of asset being sent doesn't need to be the same kind as the asset that is received. The asset kind may be exchanged a few times during a payment."

**Comment by dora-gt** [comment]
You can modify files here and make pull-requests.
Interledger is an open community, and I think this kind of problem should be solved easily by such a 
 manner.
